### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by filipnavara**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1 @@
-# test
```
Comment (filipnavara) [inline README.md:1]: I don't think this is a good change.

File: README.md [hunk 2]
```diff
@@ -1 +1 @@
...
\ No newline at end of file
+# test-pr
```
Comment (jonasfranz) [inline README.md:1]: single comment

File: README.md (hunk 1 above)
Comment (filipnavara) [inline README.md:1]: Hello

File: README.md (hunk 2 above)
Comment (jonasfranz) [inline README.md:1]: test

**Pull request closed without merging**